extensions/sql-database-projects/package.json

```diff
@@ -251,6 +260,11 @@
           "when": "view == sqlDatabaseProjectsView && viewItem == databaseProject.itemType.folder || viewItem == databaseProject.itemType.file",
           "group": "9_dbProjectsLast@2"
         },
+        {
+          "command": "sqlDatabaseProjects.openContainingFolder",
+          "when": "view == sqlDatabaseProjectsView && viewItem == databaseProject.itemType.project",
+          "group": "9_dbProjectsLast@8"
```

Comment:
This is just for my info: does the group number define the sequence of the commands in the context menu or is it the sequence followed in the code that matters?